Assignment6/MyArray.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Assignment6 {
+    public class MyArray<Data> : ICollection<Data> {
+        private Data[] _Storage;
+        public int Capacity { get; set; }
```

Comment:
```suggestion
        public int Capacity { get; }
```
You should be able to make this read-only if you want.

Reply:
Makes sense to me. Thanks!